src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -93,6 +94,20 @@ public void savesAndLoads() throws Exception {
         );
     }
 
+    @Test
+    public void saveOverwrites() throws Exception {
+        final byte[] original = "1".getBytes();
+        final byte[] updated = "2".getBytes();
+        Assume.assumeThat(updated, Matchers.not(Matchers.equalTo(original)));
```

Comment:
@olegmoz Do not use static matchers. Anyways, this assumption will never be false, so why have it?

Reply:
@paulodamaso okey, removed it

Reply:
@olegmoz Thanks!